### PR TITLE
Test that predefined poses are converted equally to and from the echo database

### PR DESCRIPTION
**Pull request by eivindsjovold**

### CODE REVIEW

File: backend/api.test/Database/Models.cs [hunk 1]
```diff
@@ -10,12 +10,63 @@ public class TestPose
...
+        }
+
+        private EchoPose ConvertPredefinedPoseToEchoPose(Position position, Orientation orientation, string frame)
+        {
+            var enuPosition = new EchoVector(position.X, position.Y, position.Z);
+            var axisAngle = ConvertOrientation(orientation);
+            return new EchoPose(enuPosition, axisAngle, "robot");
+        }
+        private static AxisAngle ConvertOrientation(Orientation orientation)
+        // This is the method used to convert predefined poses to the Angle-Axis representation used by
```
Comment (GodVenn) [inline backend/api.test/Database/Models.cs:39]: used by who?

File: backend/api.test/Database/Models.cs [hunk 2]
```diff
@@ -10,12 +10,63 @@ public class TestPose
...
+        {
+            public EchoVector Position;
+            public AxisAngle Orientation;
+            public string Frame;
+
+            public EchoPose(EchoVector position, AxisAngle orientation, string frame)
+            {
+                Position = position;
+                Orientation = orientation;
+                Frame = frame;
```
Comment (GodVenn) [inline backend/api.test/Database/Models.cs:68]: Doesn't this already exist?

Comment (eivindsjovold) [reply backend/api.test/Database/Models.cs:68]: Not to my knowledge, but it should probably

Comment (GodVenn) [reply backend/api.test/Database/Models.cs:68]: Checked, it doesn't, it's different for our other purposes so this is all good

**Review by GodVenn — APPROVED** [review]
LGTM

**Pull request merged**